README.md

```diff
@@ -177,16 +179,18 @@ This network is a pure gossip network and does not implement any form of content
 
 ## Network Specifications
 
+- [Portal Wire Protocol](./portal-wire-protocol.md)
 - [uTP over DiscoveryV5](./discv5-utp.md)
 - [State Network](./state-network.md)
     - Prior work: https://ethresear.ch/t/scalable-gossip-for-state-network/8958/4
```

Comment:
I was also thinking of removing this prior work as history-network is no longer preliminary and thus it is not that useful?